avalanche-network-runner-sdk/scripts/build.release.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -xue
+
+if ! [[ "$0" =~ scripts/build.release.sh ]]; then
```

Comment:
probably a check that also includes posibility of using . dir at start of path

Reply:
This syntax also works for `./scripts/...` 

```bash
❯ ./scripts/build.release.sh 
+ [[ ./scripts/build.release.sh =~ scripts/build.release.sh ]]
+ git submodule update --init --remote
```